scripts/services/mycroft.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Mycroft A.I. Software stack.
+After=network-online.target
```

Comment:
i dislike this one, if there is no internet i still want mycroft to start and give appropriate logs, and possibly handling it. i have some setups with cached dialog for this kind of situation

Reply:
Quick reply: network-online.target is for having the network up, but has nothing todo with the internet. This makes sure the service is started after lo, eth,wlan, etc are up.